### ISSUE TITLE
The accuracy of the training set and the test set with rbf-softmax is always 0.1

### ISSUE THREAD

**Issue by 392398741nm** [2han9x1a0release/RBF-Softmax#2]
Hello, I replace "nn.Linear" to "rbflogit.py" on MNIST. But the accuracy of the training set and the test set is always 0.1. Epoch was set to 80 and scale hyperparameter has been larged. If I use other loss functions, the net will work normally. I don't know why. Looking forward to your reply!
